brem/__init__.py

```diff
@@ -29,6 +29,7 @@
     from .brem import BasicRemoteExecutionManager
```

Comment:
need to add:
`from .brem import Windows, POSIX`
above the other imports 

This should fix the failing gh-action